lib/node_modules/@stdlib/array/base/with/benchmark/benchmark.js

```diff
@@ -0,0 +1,52 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var updateatindex = require( '@stdlib/array/base/with/lib' );
+var uniform = require( '@stdlib/random/array/uniform' );
+var bench = require( '@stdlib/bench' );
+var rand = require( '@stdlib/random/base/randu' );
+var pkg = require( '@stdlib/array/base/with/package.json' ).name;
+var value;
```

Comment:
Let's declare this variable inside the benchmark function at L34, as we don't need it in the outer scope.